src/Admin/AdminHelper.php

```diff
@@ -165,7 +165,7 @@ public function appendFormFieldElement(AdminInterface $admin, object $subject, s
 
             $modelClassName = $this->getModelClassName(
                 $admin,
-                explode('.', preg_replace('#\[\d*?]#', '', $path))
+                explode('.', preg_replace('#\[\d*?]#', '', $path) ?? '')
```

Comment:
`getModelClassName` does not support empty array, should we add a null check inside `getModelClassName` ?